library/testing/TestSDKExternalWindowGLX.cxx

```diff
@@ -0,0 +1,103 @@
+#include "engine.h"
```

Comment:
https://community.khronos.org/t/minimal-glx-opengl3-0-example/55859

No license sadly, I've reached out to the user.

Reply:
BSD-3-Clause is fine, please just add a copyright info in the file: `Copyright 2008 Arne Reiners`